app/controllers/order_controller.rb

```diff
@@ -0,0 +1,26 @@
+class OrderController < ApplicationController
```

Comment:
@domenic99 why is it `OrderController`, not `OrdersController`? We will have a lot of orders, not a one order :stuck_out_tongue_winking_eye: